Week-13/homework-13/homework13.js

```diff
@@ -0,0 +1,162 @@
+let quizData = [
+  {
+    question: 'What does HTML stand for?',
+    answers: [
+      { text: 'Hyper Text Markup Language', correct: true },
+      { text: 'High Text Markup Language', correct: false },
+      { text: 'Hyper Tabular Markup Language', correct: false },
+      { text: 'None of these', correct: false },
+    ],
+  },
+  {
+    question: 'Which language runs in a web browser?',
+    answers: [
+      { text: 'Java', correct: false },
+      { text: 'C', correct: false },
+      { text: 'Python', correct: false },
+      { text: 'JavaScript', correct: true },
+    ],
+  },
+  {
+    question: 'What does CSS stand for?',
+    answers: [
+      { text: 'Central Style Sheets', correct: false },
+      { text: 'Cascading Style Sheets', correct: true },
+      { text: 'Cascading Simple Sheets', correct: false },
+      { text: 'Cars SUVs Sailboats', correct: false },
+    ],
+  },
+  {
+    question: 'What year was JavaScript launched?',
+    answers: [
+      { text: '1996', correct: false },
+      { text: '1995', correct: true },
+      { text: '1994', correct: false },
+      { text: 'none of the above', correct: false },
+    ],
+  },
+  {
+    question: 'Which tool can you use to ensure code quality?',
+    answers: [
+      { text: 'Angular', correct: false },
+      { text: 'jQuery', correct: false },
+      { text: 'RequireJS', correct: false },
+      { text: 'ESLint', correct: true },
+    ],
+  },
+];
+
+//For video referening --> myNames = videoNames
+//quizData = questions
+//questionElement = questionElement
+//answerButtons = answerButtons
+//nextButton = nextButton
+
+// Selecting the question, question number, answer buttons, and next button
+let questionElement = document.querySelector(`[data-component="question"]`);
+let questionNumber = document.querySelector(
+  `[data-component="questionNumber"]`
+);
+let answerButtons = document.querySelector(`[data-component="answers"]`);
+let nextButton = document.querySelector(`[data-component="next"]`);
+
+// Setting some index for this medthod?
+let currentQuestionIndex = 0;
+let score = 0;
+
+//Im guessing a function when starting the page, also adds "NeXT" into nextButton
+function startQuiz() {
+  currentQuestionIndex = 0;
+  nextButton.innerHTML = 'Next';
+  score = 0;
+  showQuestion();
+}
+
+//Creating some function that would load corresponding question number and question
+function showQuestion() {
+  resetState();
+
+  let currentQuestion = quizData[currentQuestionIndex];
+  let questionNumber = currentQuestionIndex + 1;
+
+  questionElement.innerHTML =
+    'Question ' + questionNumber + '. ' + currentQuestion.question;
+
+  //Adding the answers as buttons
+  currentQuestion.answers.forEach((answer) => {
+    let button = document.createElement('button');
+    button.innerHTML = answer.text;
+    button.classList.add('btn');
```

Comment:
You can chain multiple class in here
```
button.classList.add('btn', 'btn-outline-secondary', 'd-block')
```

Reply:
I did try to do this on my first attempt, but it didn't work. After working on it now, somehow it did....
Thanks for that